src/main/java/uk/ac/cam/cl/dtg/util/mappers/UnimplementedMappingException.java

```diff
@@ -0,0 +1,16 @@
+package uk.ac.cam.cl.dtg.util.mappers;
+
+public class UnimplementedMappingException extends RuntimeException {
+  public UnimplementedMappingException() {
```

Comment:
I would consider removing this constructor, and the one below, since they're not used. If we ever have the need for them, we can easily add them.